lib/Mojo/ACME.pm

```diff
@@ -44,7 +46,7 @@ has ua => sub {
 sub check_all_challenges {
   my ($self, $cb) = (shift, pop);
   my @pending = $self->pending_challenges->each;
-  Mojo::IOLoop->delay(
+  Mojo::IOLoop::Delay->new->steps(
```

Comment:
Isn't the point of using `Mojo::IOLoop::Delay` that you don't have to change the calling code? 
 
(You know, since `Mojo::IOLoop::Delay` just shims `delay` back into `Mojo::IOLoop`)